addon-scheduler/src/main/java/org/vaadin/stefan/fullcalendar/Resource.java

```diff
@@ -279,8 +278,13 @@ protected JsonObject toJson() {
         jsonObject.put("title", JsonUtils.toJsonValue(getTitle()));
         jsonObject.put("eventColor", JsonUtils.toJsonValue(getColor()));
         
-        if(getBusinessHours() != null)
-        	jsonObject.put("businessHours", getBusinessHours().toJson());
+        if(getBusinessHoursArray() != null) {
```

Comment:
Also check for a non empty array here to prevent sending empty arrays all the time, since the array might be empty when set via constructor.

Reply:
Added the check for empty array